internal/kafka/internal/workers/kafka_mgrs/kafkas_mgr.go

```diff
@@ -440,7 +440,7 @@ func (k *KafkaManager) updateExpiresAtBasedOnQuotaEntitlement(kafka *dbapi.Kafka
 		}
 
 		// set expires_at to now + grace period days
```

Comment:
the comment needs to be updated as well?

Reply:
yes. I will update it. I also need to add code to remove expiration if quota is now > 0